.github/CHANGELOG.md

```diff
@@ -19,10 +22,10 @@
 
 ### Breaking changes
 
-* Add `tests_gpu.yml` workflow to test the Lightning-Kokkos backend with CUDA-12. 
+* Add `tests_gpu.yml` workflow to test the Lightning-Kokkos backend with CUDA-12.
```

Comment:
can you specify its name with `kokkos_gpu`?

Reply:
Since this is the name of the file at that point, I think it is better not. Otherwise, we would have to search and replace strings in the whole CHANGELOG every time we rename a file.